### PR TITLE
Use Pipenv to address the dependency of Flake8

### PR DESCRIPTION
**Pull request by yykamei**
Like #99, I want to install Flake8 via Pipenv.

### CODE REVIEW

**Comment by yykamei** [comment]
We should install Pipenv in devon_rex. I'm addressing it with sider/devon_rex#48.

File: images/flake8/Pipfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[packages]
+flake8 = "==3.7.7"
```
Comment (ybiquitous) [inline images/flake8/Pipfile]: [question] Is `3.7.4` installed when seeing a CircleCI build? ⬇️ 

https://circleci.com/gh/sider/runners/7961

image

Comment (yykamei) [reply images/flake8/Pipfile]: This is the Python runtime version generated by `pyenv whence`. However, the expected output is `2.7.16\n3.7.4\n` :thinking: 
I'll fix Dockerfile.python.erb

**Comment by ybiquitous** [comment]
Dependabot supports now `Pipenv`!

https://dependabot.com/python/

File: images/Dockerfile.python.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+COPY images/<%= analyzer %>/Pipfile ${RUNNER_USER_HOME}/<%= analyzer %>_Pipfile
+RUN cd ${RUNNER_USER_HOME} && \
+    mkdir <%= analyzer %> && \
+    mv <%= analyzer %>_Pipfile <%= analyzer %>/Pipfile && \
+    cd <%= analyzer %> && \
+    for version in "${PYTHON2_VERSION}" "${PYTHON3_VERSION}"; do \
+      pyenv global ${version} && \
+      pipenv install --system --skip-lock && \
+      pyenv rehash && \
+      echo "In '$(python --version 2>&1)', the version of the analyzer '<%= analyzer %>' is ..." && \
```
Comment (yykamei) [inline images/Dockerfile.python.erb]: Python 2 writes its version to STDERR, so I used redirection here.

File: images/flake8/Pipfile (hunk 1 above)
Comment (ybiquitous) [reply images/flake8/Pipfile]: Looks good! 👍 

image

**Pull request merged**